elastic/kibana
# [Infrastructure UI] Kubernetes Pods correctly show Network chart

## 📓 Summary

Closes #129239 

This issue was caused by a Validation error when creating the tsvb model for the `podNetworkTraffic` metric series.
It was basically referencing an aggregation property that didn't exist, caused by different property names matching.

## 🧪 Testing

- Navigate to Inventory
- Select `Show` -> `Kubernetes Pods`
- Select any available pod and click on Kubernetes pod metrics to open the pod details view
- Verify the Network chart is available and matches the **Inbound (RX)** and **Outbound (TX)** values.

https://user-images.githubusercontent.com/34506779/227875972-07aa35e1-eb0c-480a-8500-342de6ffad80.mov
